src/components/LoanCollections/KivaClassicMultiCategoryCarousel.vue

```diff
@@ -80,6 +89,23 @@ export default {
 	mounted() {
 		this.fetchLoanChannel();
 	},
+	created() {
+		// this experiment is assigned in experimentPreFetch.js
+		const urgencyExperiment = this.apollo.readFragment({
+			id: 'Experiment:lend_urgency',
+			fragment: experimentVersionFragment,
+		}) || {};
+		this.isUrgencyExpVersionShown = urgencyExperiment.version === 'shown';
```

Comment:
@eddieferrer if you really need to confine this to the homepage you'll need to check against the route here too. Otherwise, this exp could be active on landing pages that use this Contentful based component.

Reply:
Good point, thank you! 👍